File: app/src/main/java/com/apache/fastandroid/ui/fragment/setting/SettingFragment.java
package com.apache.fastandroid.ui.fragment.setting;

import android.app.Activity;
import android.os.Bundle;
import android.preference.PreferenceFragment;
import android.support.annotation.Nullable;

import com.apache.fastandroid.R;
import com.tesla.framework.ui.activity.BaseActivity;
import com.tesla.framework.ui.activity.FragmentContainerActivity;

/**
 * Created by jerryliu on 2017/6/24.
 */

public class SettingFragment extends PreferenceFragment {
    public static void launch(Activity from){
        FragmentContainerActivity.launch(from,SettingFragment.class,null );
    }

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.fragment_setting);
    }

    @Override
    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);
        BaseActivity activity = (BaseActivity) getActivity();
        activity.getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        activity.getSupportActionBar().setDisplayShowHomeEnabled(false);
        activity.getSupportActionBar().setTitle("设置");
    }
}

File: app/src/main/java/com/apache/fastandroid/setting/SettingFragment.kt
package com.apache.fastandroid.setting

import com.tesla.framework.ui.fragment.preference.PreferenceLoaderFragment

/**
 * Created by Jerry on 2022/3/16.
 */
class SettingFragment : PreferenceLoaderFragment() {

    private lateinit var preferenceLoader: SettingsPreferenceLoader


    override fun loadPreferences() {
        preferenceLoader = SettingsPreferenceLoader(this)
        preferenceLoader.loadPreferences()
    }

}